spec/models/family_spec.rb

```diff
@@ -1,8 +1,18 @@
 require "rails_helper"
 
 describe Family do
-  describe "validations" do
+  context "validations" do
     it { is_expected.to validate_presence_of(:last_name) }
     it { is_expected.to validate_presence_of(:capability) }
   end
+
+  context "by_last_name" do
```

Comment:
Change this line to: `describe ".by_last_name" do`
Because we use describe for methods we test. And use `.` to indicate it is a class method (rather than `#` for instance method)

Reply:
Done in sha `4f9103b`